### PR TITLE
Fixed typo in lit-word.adoc. Revised Literal Syntax in get-word.adoc

### PR DESCRIPTION
**Pull request by gltewalt**

### CODE REVIEW

File: en/datatypes/email.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+= Email! datatype
+:toc:
+:numbered:
+
+== Abstract
+
+`Email!` values allow for directly expressing email addresses. `foo@bar.net`
```
Comment (greggirwin) [inline en/datatypes/email.adoc]: Add "e.g." before example at end of sentence?

File: en/datatypes/email.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+>> b: to email! 'bar
+== bar
+
+>> type? b
+== email!
+```
+
+== Literal syntax
+
+The `@` symbol, and zero or more additional characters.
```
Comment (greggirwin) [inline en/datatypes/email.adoc]: Let's say there have to be chars before the `@`, even though the lexer allows them not being there today. That syntax may become a `ref!` type.

File: en/datatypes/get-word.adoc [hunk 3]
```diff
@@ -37,9 +37,9 @@ A `get-word!` value retieves an unevaluated word value from the context to which
 
 == Literal syntax
 
-A colon `:` followed by one or more Unicode characters, or ASCII characters from the set:  `! & ' * + - . < = > ? _ ` | ~` 
+A colon `:` followed by One or more characters from the entire Unicode range, including punctuation characters from the ASCII subset: `! & ' * + - . < = > ? _ | ~``
```
Comment (greggirwin) [inline en/datatypes/get-word.adoc]: Reason for capitalizing "One"?

Comment (gltewalt) [reply en/datatypes/get-word.adoc]: Typo from Copy/Paste

**Pull request merged**